api-auto-registration/api-curation.hbs.md

```diff
@@ -0,0 +1,127 @@
+# API Curation (alpha)
+
+This topic describes how you can use API Curation to expose several standalone APIs as one API.
+
+To unlock the maximum power of API curation, we strongly recommend to use a supported route provider.
+Without this setup, the generated API spec will not have a functional server URL set for testing the
+curated API. And you will need to manually create the routing resources to route traffic to each
+referenced API to match on the aggregated API spec. With the route provider integration, all of these
+routing concern will be taken care of by us automatically.
+
+A successful API curation requires the following as of today:
+
+1. (Optional) Spring Cloud Gateway for Kubernetes is installed.
+2. API Auto Registration is installed or updated with `route_provider.spring_cloud_gateway.enabled`
+   set accordingly.
+3. One or more `APIDescriptor`s that are in the ready state.
+4. (Optional) A `SpringCloudGateway` resource is created with the matching `groupId` and `version`.
```

Comment:
Can we group all the optional SCG related steps at the top? Also we can doubly emphasize (Optional but recommended)